ci-operator/step-registry/storage/create/csi-gcp-filestore/storage-create-csi-gcp-filestore-commands.sh

```
@@ -19,6 +19,18 @@ export STORAGECLASS_LOCATION=${SHARED_DIR}/filestore-sc.yaml
 export MANIFEST_LOCATION=${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
 export CONNECT_MODE="DIRECT_PEERING"
 
+# Version comparison functions using sort -V
+# Copied from other test steps
+function version_ge() {
+  # Returns 0 (true) if $1 >= $2
+  [[ "$1" == "$2" ]] && return 0
+  [[ "$(printf '%s\n' "$2" "$1" | sort -V | head -n1)" == "$2" ]]
+}
+TEST_VOLUME_SIZE="1Ti"
+if version_ge "${ocp_version}" "4.22"; then
```

Shellcheck (0.11.0)

[warning] ocp_version is referenced but not assigned.

(SC2154)